docs/_static_html/conf/eu/2015/news/announcing-eu-presentations/index.html

```diff
@@ -175,9 +175,9 @@ <h2>Presentations<a class="headerlink" href="#presentations" title="Permalink to
 <div class="section" id="what-s-next">
 <h2>What’s Next<a class="headerlink" href="#what-s-next" title="Permalink to this headline">¶</a></h2>
 <p>If you haven’t purchased your ticket yet, you still have some time! You can
-get your tickets <a class="reference external" href="http://writethedocs.org/conf/eu/2015/#tickets">here</a>.</p>
+get your tickets <a class="reference external" href="https://writethedocs.org/conf/eu/2015/#tickets">here</a>.</p>
```

Comment:
These should probably migrate to `www.` as well, if we care about not having a redirect?

Reply:
Done